app/controllers/spree/admin/orders_controller.rb

```diff
@@ -78,7 +78,7 @@ def fire
       end
 
       def resend
-        Spree::OrderMailer.confirm_email_for_customer(@order.id, true).deliver
+        Spree::OrderMailer.confirm_email_for_customer(@order.id, true).deliver_now
```

Comment:
wait, this won't enqueue an async job, right? IMO we really need to replace all these `deliver_now` in controllers, models and service objects to `deliver_later`.

Reply:
Currently whenever we call `#deliver`, it's logging a deprecation message, then silently calling `#deliver_now` under the hood. So this change just makes the current behavior more explicit.

Totally agree we should assess various places where we send mails and probably change most of them to `#deliver_later` :100:

Reply:
Do we want to do that in this PR? It would require various spec changes as well. I don't mind if we do it here or a separate PR...

Reply:
I'd take the chance now that we're doing it and Rails is saving us under the hood :pray:

Reply:
We might need to think about running `DelayedJob` with 2 workers if we're dumping a lot more jobs onto it. Eg: if it ever gets stuck processing a single job that takes a fair amount of time, it will hold up all emails until it's finished...

Reply:
> in controllers, models and service objects

So... everywhere? :trollface:

Reply:
all but jobs :joy:

Reply:
At some point it might be nice to switch from Memcached and DelayedJob to Redis and Sidekiq, buuuuut.... not today.

Reply:
yes, we both share the goal and the fear to touch the infra so much :joy: